apis/python/tests/test_context.py

```diff
@@ -137,7 +137,7 @@ def test_replace_config_after_construction():
 def test_malformed_concurrency_config_value():
     import numpy as np
 
-    with pytest.raises(tiledbsoma.SOMAError):
+    with pytest.raises((tiledbsoma.SOMAError, RuntimeError)):
```

Comment:
I'll help you fix this -- already done on https://github.com/single-cell-data/TileDB-SOMA/pull/2963

* https://github.com/single-cell-data/TileDB-SOMA/blob/c08b5cf544040a9601fa8c19496ff4af6c7f2c8a/apis/python/src/tiledbsoma/pytiledbsoma.cc#L36-L45
* https://github.com/single-cell-data/TileDB-SOMA/blob/c08b5cf544040a9601fa8c19496ff4af6c7f2c8a/apis/python/src/tiledbsoma/soma_array.cc#L348-L352 (one example callsite)

Reply:
Also
https://pybind11.readthedocs.io/en/stable/advanced/exceptions.html

Reply:
Broader issue - filed as sc-54978